lib/jobs/dell-wsman-RAID.js

```diff
@@ -56,10 +56,22 @@ function DellWsmanRAIDFactory(
 
     DellWsmanRAIDJob.prototype._handleSyncRequest = function() {
         var self = this;
-        return self.checkOBM('SCP RAID operation')
-        .then(function(obm){
+        if(self.options.ipAddress === '' || self.options.username === '' ||
+            self.options.password === ''){
```

Comment:
Your schema doesn't restrict type of those ipAddress/username/password to be string, you should consider cases that those parameters to be null and undefined as well. 
Expression like (Job.Dell.Wsman.RAID || self.options.username ||  self.options.password) should be better for you.

Reply:
The type of ipAddress/username/password is string, which defined in schema.

Reply:
If you are mentioning dell-wsman-RAID.json, I didn't see that.

Reply:
OK, I see you update schema in this PR as well.